python/llm/dev/benchmark/all-in-one/run-hbm.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+source bigdl-llm-init
+
+sockets_num=$(lscpu | grep "Socket(s)" | awk -F ':' '{print $2}')
+cores_per_socket=$(lscpu | grep "Core(s) per socket" | awk -F ':' '{print $2}')
+numa_nodes=$(lscpu | grep "NUMA node(s)" | awk -F ':' '{print $2}')
+# Multiply by 2 to avoid an float result in HBM flat mode that the NUMA count twice and it will be divided later.
+cores_per_numa=$(($sockets_num * $cores_per_socket * 2 / $numa_nodes))
+
+if [ "${numa_nodes}" -eq 16 ]; then
+    #HBM flat SNC-4 mode, Confirm that there are 8 HBM memory nodes and 8 DRAM memory nodes through "numactl -H"
+    #0-7 is DRAM memory node, 8-15 is HBM node
+    export OMP_NUM_THREADS=${cores_per_numa}
+    echo "OMP_NUM_THREADS: ${cores_per_numa}"
+    echo "HBM SNC4 mode"
+    last_cpu_index=$(($OMP_NUM_THREADS - 1))
+    numactl -C 0-$last_cpu_index -p 8 python $(dirname "$0")/run.py
```

Comment:
hard coded 8 is not a good idea for larger models. Because we have only 16GB HBM on this node.